### ISSUE TITLE
Provide HTML Serializer children as string

### ISSUE THREAD

**Issue by angeloashmore** [prismicio/prismic-helpers#25]
### Is your feature request related to a problem? Please describe.

When writing HTML Serializers, the `children` parameter is an array of strings. Whenever it is used in the return value, it must always be converted into a single string using something like `children.join('')`. This can be verbose.

### Describe the solution you'd like

Since the return value must always be a string, could the HTML Serializer be typed so `children` is already a flat string?

Before:
```ts
const htmlSerializer = {
  heading1: ({ children }) => `<h1>${children.join('')}</h1>,
}
```

After:
```ts
const htmlSerializer = {
  heading1: ({ children }) => `<h1>${children}</h1>,
}
```

### Describe alternatives you've considered

None.

### Additional context

This changes the type signature of a Rich Text Serializer so there may be type conflicts. `children` is expected to always be an array in `@prismicio/richtext`. We may need to make `children` a generic that can be overridden by libraries (@lihbr I think you may have done this in the past… sorry if I reverted that!).

**Comment by lihbr** [comment]
All up for it, can we try considering what kind of use case might actually prefer an array of string for children? :thinking: Just want to make sure we're not closing opportunities :)

**Comment by lihbr** [comment]
@angeloashmore do we want to move on with that while we're still in beta?

**Comment by angeloashmore** [comment]
@lihbr Yes, let's address this before the full release. I'll give it a try.

**Comment by lihbr** [comment]
Awesome, should be a quick one, let me know if you need me ☺️

**Comment by lihbr** [comment]
Closed in #29
